ds/data/processed/hebci/contract_research_sme.nuts3.csv

```diff
@@ -0,0 +1,446 @@
+year,nuts_id,nuts_year_spec,contract_research_sme
+2014,UKC12,2013,181000.0
+2014,UKC14,2013,410000.0
+2014,UKC22,2013,635000.0
+2014,UKC23,2013,0.0
```

Comment:
are these zeroes actual zeroes or `null` coerced to zero?

Reply:
These are true zeros